projects/novo-elements/src/elements/data-table/data-table.source.ts

```diff
@@ -16,7 +16,7 @@ export class DataTableSource<T> extends DataSource<T> {
   private totalSet: boolean = false;
 
   get totallyEmpty(): boolean {
-    return this.total === 0;
+    return this.total === 0 && this.current === 0;
```

Comment:
might not need this, since it's a direct check for 0 and this.total will be null